addons/web/controllers/main.py

```diff
@@ -1082,7 +1082,7 @@ def _render_template(self, **d):
         def load(template_name, options):
             return (html.fragment_fromstring(templates[template_name]), template_name)
 
-        return QWeb()._render(html.document_fromstring(template), d, load=load)
+        return IrQWeb()._render(html.document_fromstring(template), d, load=load)
```

Comment:
@rco-odoo This change is insufficient because it is necessary to instantiate `ir.qweb` and not the class python. Do you have a clean way?